hypervisor.go

```diff
@@ -127,6 +127,9 @@ type HypervisorConfig struct {
 	// ImagePath is the guest image host path.
 	ImagePath string
 
+	// BiosPath is the bios host path
+	BiosPath string
```

Comment:
`FirmwarePath` please.

Reply:
Q. why firmware? 

I though bios was right, because the firmware is the binary that allows to the bios to access to different resources present in the host like kernel, cmdline, initrd, etc, through DMA [1] [2]

[1] https://github.com/bonzini/qboot/blob/master/fw_cfg.c#L186-L228
[2] https://github.com/clearcontainers/qemu/blob/master/hw/i386/pc_sysfw.c#L104-L174

Reply:
@devimc They're essentially the same, bios being a term that's generally used for legacy firmware implementations. firmware is more generic.

Reply:
ohh ok, make sense